### PR TITLE
A few minor tweaks

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

File: configuration.rst [hunk 1]
```diff
@@ -344,7 +344,7 @@ files are always in sync.
 
     When you :ref:`install an existing Symfony project <install-existing-app>`, you
     will need to create the ``parameters.yml`` file using the committed ``parameters.yml.dist``
-    file as a reference. To help with this, after you run ``composer install``, a
+    file as a reference. To help with this, after you run ``$ composer install``, a
```
Comment (javiereguiluz) [inline configuration.rst]: For commands displayed inside a paragraph we don't include the leading `$`, only for proper console blocks.

**Review by javiereguiluz — APPROVED** [review]
👍 

Thanks @bocharsky-bw

File: configuration.rst (hunk 1 above)
Comment (bocharsky-bw) [inline configuration.rst]: Good to know, I reverted it

**Comment by wouterj** [comment]
👍 

status: reviewed

**Comment by wouterj** [comment]
Perfect PR, thanks! Lots of very good catches. (btw, I personally always use config:dump, which also works because of the abbr support of the Console component)

**Pull request merged**